Eradication/VectorPopulation.cpp

```diff
@@ -2846,49 +2795,39 @@ namespace Kernel
                                                    VectorCohortVector_t* pMigratingQueue,
                                                    VectorCohortCollectionAbstract& rQueue )
     {
+
         Gender::Enum             human_gender_equivalent = m_pMigrationInfoVector->ConvertVectorGender( vector_gender );
-        const std::vector<suids::suid>& reacheable_nodes = m_pMigrationInfoVector->GetReachableNodes( human_gender_equivalent );
-        std::vector<float>            fraction_traveling = m_pMigrationInfoVector->GetFractionTraveling( vector_gender, 0 );
-        // we could use FractionTraveling to also set the GetTotalFemale and Male, but for that we'd have to keep track of male totals with a new parameter
+        const std::vector<suids::suid>  reacheable_nodes = m_pMigrationInfoVector->GetReachableNodes( human_gender_equivalent );
+        std::vector<float>            fraction_traveling = m_pMigrationInfoVector->GetFractionTraveling( vector_gender, 0 ); // index = 0 is the rate for non-allele travel
```

Comment:
If we do the GetFractionTraveling() I suggest below, should not need this one here.

Reply:
I don't like getting it over and over again when we don't actually need to :/ I kept tthe "IsTravelingByAlleles"